lib/stronger_parameters/parameters.rb

```diff
@@ -190,11 +191,16 @@ module ControllerSupport
       # TODO: this is not consistent with the behavior of raising ActionController::UnpermittedParameters
       # should have the same render vs raise behavior in test/dev ... see permitted_parameters_test.rb
       rescue_from(StrongerParameters::InvalidParameter) do |e|
-        render plain: e.message, status: :bad_request # uncovered
+        if request.format.to_s.include?('json')
```

Comment:
FYI avoided use to `respond_to do |format|` since it is included in ActionController::API by default.